src/main/java/no/unit/nva/bare/CreateAuthorityHandler.java

```diff
@@ -67,34 +65,33 @@ protected GatewayResponse createAuthorityOnBare(String name) {
         GatewayResponse gatewayResponse = new GatewayResponse();
         AuthorityConverter authorityConverter = new AuthorityConverter();
         BareAuthority bareAuthority = authorityConverter.buildAuthority(name);
-        try (CloseableHttpResponse response = bareConnection.createAuthority(bareAuthority)) {
+        try {
+            HttpResponse<String> response = bareConnection.createAuthority(bareAuthority);
             System.out.println("response (from bareConnection)=" + response);
-            if (response.getStatusLine().getStatusCode() == Response.Status.CREATED.getStatusCode()
-                || response.getStatusLine().getStatusCode() == Response.Status.OK.getStatusCode()) { //201
-                try (Reader streamReader = new InputStreamReader(response.getEntity().getContent())) {
-                    BareAuthority createdAuthority = new Gson().fromJson(streamReader, BareAuthority.class);
-                    if (Objects.nonNull(createdAuthority)) {
-                        final Authority authority = authorityConverter.asAuthority(createdAuthority);
-                        List<Authority> authorities = new ArrayList<>();
-                        authorities.add(authority);
-                        Type authorityListType = new TypeToken<ArrayList<Authority>>(){}.getType();
-                        Gson gson = new Gson().newBuilder().setPrettyPrinting().create();
-                        gatewayResponse.setBody(gson.toJson(authorities, authorityListType));
-                        gatewayResponse.setStatusCode(Response.Status.OK.getStatusCode());
-                    } else {
-                        System.out.println(String.format(COMMUNICATION_ERROR_WHILE_CREATING, name));
-                        gatewayResponse.setErrorBody(String.format(COMMUNICATION_ERROR_WHILE_CREATING, name));
-                        gatewayResponse.setStatusCode(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode());
-                    }
+            if (response.statusCode() == Response.Status.CREATED.getStatusCode()
```

Comment:
The status management can also be handled in the fluent API

Reply:
I do not really get this comment but it is out of scope of the java 11 upgrade pull request.